swagger2.cabal

```diff
@@ -93,11 +97,11 @@ test-suite spec
     Data.Swagger.Schema.ValidationSpec
   default-language: Haskell2010
 
-test-suite doctest
-  build-depends:    base, doctest, Glob
+test-suite doctests
+  build-depends:    base, doctest, Glob, QuickCheck
```

Comment:
Made a PR. https://github.com/phadej/cabal-doctest/pull/16 and added a link to where it will land.